v3/src/components/graph/adornments/univariate-measures/standard-error/standard-error-adornment-model.ts

```diff
@@ -26,7 +36,7 @@ export const StandardErrorAdornmentModel = UnivariateMeasureAdornmentModel
       const caseValues = self.getCaseValues(attrId, cellKey, dataConfig)
       // If there are less than two values, the adornment should not render.
       if (caseValues.length < 2) return
-      return self.numStErrs * Number(std(caseValues)) / Math.sqrt(caseValues.length)
+      return this.numStErrs * Number(std(caseValues)) / Math.sqrt(caseValues.length)
```

Comment:
I think it's preferred and would keep things more consistent to use `self` instead of `this` here, and move `computeMeasureValue` to a separate `views` block. Using `this` should work OK for now, but it could potentially cause problems in the future as things get changed and possibly shifted around. I don't think it's a huge problem, though, if you really want to keep this grouped with the other methods here.